backend/setup_env.py

```diff
@@ -0,0 +1,174 @@
+"""
+Script built on dotenv to set up the required environment variables
+"""
+
+import argparse
+from os import write
+from pathlib import Path
+from typing import Optional
+
+import dotenv
+
+# Assumes this file is in `/backend/setup_env.py`
```

Comment:
thoughts on making this a root folder script? since its not just backend stuff

Reply:
Is our venv initialised to root or backend?

Reply:
the venv folder should be living in your root

Reply:
Okay, sounds good